test/Index/objc_conformances.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: %target-swift-ide-test -print-indexed-symbols -source-filename %s
```

Comment:
I think this needs `REQUIRES: objc_interop`.
Also if `import Foundation` is required to be in the test case, I'd suggest to change to `// RUN: %target-swift-ide-test(mock-sdk: %clang-importer-sdk)`